saleor/warehouse/utils.py

```diff
@@ -0,0 +1,39 @@
+from collections import namedtuple
+
+from ..plugins.manager import get_plugins_manager
+
+
+class ProductVariantStockWebhookTrigger:
+    _stock_data = namedtuple(
+        "_stock_data", ("stock", "is_created", "prev_quantity", "current_quantity")
+    )
+
+    def __init__(self):
+        self._stocks_data = []
+        self._plugins_manager = get_plugins_manager()
```

Comment:
Always take manager from context.
Also, not sure if Warehouse/utils is a good place for this. I understand that webhook is called when we make changes in stock, but we raise an event dedicated to ProductVariant. IMHO the logic for raising these webhooks should be in the product module.
I really don't think that we get here any prons that this is a class